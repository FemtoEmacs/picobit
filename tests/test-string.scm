;; tests with strings

(define (foo x)
  (+ (string-length x) 1))

(display (foo "abc"))
