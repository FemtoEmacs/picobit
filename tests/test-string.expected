:0080000080
:10800000FBD72102811D400081124000810E4000FB
:10801000810C4000810D4000810A400080650119FB
:10802000804200028027011A8027011B8024011C46
:108030008071011E806E011F806700028027012071
:10804000807701218073012280240002806A00026F
:108050008073012380670124806601118040011331
:1080600080400114803201158027011680660117B7
:108070008069011880780002807D010F8066011000
:108080008070010B8078010B02B401895002B401A9
:108090008351A109B00165B582EED80120DAB89507
:1080A0004121DBB6800221CC9A21CE2161EE202233
:1080B000CFB67200EF20B6800120DA9EA02607E638
:1080C00020B558EEA02607E600EF20C0B8BF20B5C7
:1080D00082B648012004C8B8AEA03121B5A402B5CB
:1080E00083CDDCEF02A034210EC5C122CD2102CD0B
:1080F000B400F47101210EC89220EF210E406221DC
:10810000B662010421C2EF2002B559B75520CCB8A0
:10811000B8A02C07E620CEB51FEE20CFB6800120F8
:10812000D296A02D07E600EF20CC9DA02407E620E4
:10813000CEB505EE20CFB666A106B0009BEE20B00E
:1081400000B8EEA02D07E600EF20D995A107B100F9
:108150009B20F09C2095A105B1009BA104B1009B40
:10816000A108B1009B0120B58305C1EF0120DBB65A
:1081700080012004B6800220CC9720CF2205C1B612
:108180007521EF012007E600EF022004C821230239
:10819000CDCDB4019671012204C822242302CDCD95
:1081A000CDB401A5710123B8DC21B0010223222640
:1081B00002CDCDCDB401B8710122B8C621B0010203
:1081C00021242602CDCDCDB401CB7101202224265D
:1081D00002CDCDCDCDB401D970002121C42325021B
:1081E000CDCDB401E6710122B893219220EF2298FF
:1081F000019420B1010220EF21B87BB77500B86E61
:08820000B76A21B73B21B72545
:00000001FF
